apm-sniffer/apm-sdk-plugin/rocketMQ-4.x-plugin/pom.xml

```diff
@@ -30,7 +30,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <rocketmq-client.version>4.1.0-incubating</rocketmq-client.version>
+        <rocketmq-client.version>4.3.1</rocketmq-client.version>
```

Comment:
Why bump up this version? Does this mean you break the support list?

Reply:
It could be reverted, just want to use the release version, rather than the `incubation` version.

Reply:
https://mvnrepository.com/artifact/org.apache.rocketmq/rocketmq-client